### PR TITLE
vboxmanage-*: add page

### PR DESCRIPTION
**Pull request by HOIg3r**
- [x] The page(s) are in the correct platform directories: `common`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/vboxmanage-export.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Export virtual machines to a virtual appliance (iso) or a cloud service.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-export>.
+
+- Clone the specified VM:
+
+`VBoxManage clonevm {{vm_name}}`
+
+- Set a new name for the new VM:
+
+`VBoxManage clonevm {{vm_name}} --name {{exemple_name}}`
```
Comment (gutjuri) [inline pages/common/vboxmanage-export.md:12]:
```suggestion
`VBoxManage clonevm {{vm_name}} --name {{example_name}}`
```

File: pages/common/vboxmanage-export.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Indicate the folder where the configuration of the new VM is gonna be stored:
```
Comment (gutjuri) [inline pages/common/vboxmanage-export.md:14]:
```suggestion
- Indicate the folder where the configuration of the new VM will be stored:
```

File: pages/common/vboxmanage-import.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# vboxmanage-import
+
+> Imports a virtual machine (VM) previously exported or created.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-import>.
+
+- Import a VM from a OVF or OVA file:
```
Comment (gutjuri) [inline pages/common/vboxmanage-import.md:6]:
```suggestion
- Import a VM from an OVF or OVA file:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thank you! I have added some suggestions to fix some typos.

File: pages/common/vboxmanage-clonevm.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Creates a clone of an existing virtual machine (VM).
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-clonevm>.
+
+- Clone the specified VM:
+
+`VBoxManage clonevm {{vm_name}}`
+
+- Specifie a new name for the new VM:
+
+`VBoxManage clonevm {{vm_name}} --name {{exemple_name}}`
```
[the remark this answers is not in the document]
Comment (HOIg3r) [reply pages/common/vboxmanage-clonevm.md:12]: I've tested and read up on the subject and the 2 are equivalent - you can use quotes or not.

**Review by gutjuri — APPROVED** [review]
Thanks!

**Pull request merged**